src/Geta.404Handler/Controllers/NotFoundRedirectController.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Copyright (c) Geta Digital. All rights reserved.
```

Comment:
It seems that some file encoding has changed. Please make sure that it is the same as before.
As I remember, it should be UTF-8 with BOM.